### PR TITLE
Improving AgentConfig and logging

### PR DESCRIPTION
**Pull request by matt-raffel-kiva**
Signed-off-by: Matt Raffel <mattr@kiva.org>

### CODE REVIEW

File: src/manager/k8s.service.ts [hunk 1]
```diff
@@ -33,8 +33,6 @@ export class K8sService implements IAgentManager {
     }
 
     private async createPod(config: AgentConfig): Promise<any> {
-        const inboundTransportSplit = config.inboundTransport.split(' ');
```
Comment (matt-raffel-kiva) [inline src/manager/k8s.service.ts:36]: there are some other process.env....statements in the k8s.service.ts that I did not change.  I felt these were specific to k8s so it was ok to leave them here.

**Review by jeffk-kiva — APPROVED** [review]
Looks great! Glad to see the process.env moved to the config class. :)

File: src/manager/k8s.service.ts (hunk 1 above)
Comment (jsaur) [reply src/manager/k8s.service.ts:36]: Yep that makes sense

**Review by jsaur — APPROVED** [review]
Looks good 👍

**Pull request merged**